moveit_planners/ompl/ompl_interface/test/test_state_space.cpp

```diff
@@ -167,6 +167,34 @@ TEST_F(LoadPlanningModelsPr2, StateSpaceCopy)
     EXPECT_TRUE(robot_state.distance(robot_state2) < 1e-12);
   }
 
+  // repeat the above test with a different method to copy the state to OMPL
+  for (int i = 0; i < 10; ++i)
+  {
+    // create two equal states
+    moveit::core::RobotState robot_state2(robot_state);
+    EXPECT_TRUE(robot_state.distance(robot_state2) < 1e-12);
```

Comment:
```suggestion
    EXPECT_LT(robot_state.distance(robot_state2), 1e-12);
```

I see that the old code in this file also does the comparison this way.  The advantage of using `EXPECT_LT` is that it will print the value when it fails instead of just saying that it was false.  Also, since 1e-12 seems to be being used as an epsilon you could put a constant at the top of the file and replace all the uses with the constant.

This is more of just a nit as you are following the style already in this test.

Reply:
I changed it to `std::numeric_limits<double>::epsilon();`, but used a variable name that is a bit more reasonable in length.

The tests seem to work, but I decreased the test tolerance from `1e-12` to `1e-16` ish. I'm a bit worried this will cause random test failures because the distance calculation probably contains summations. Maybe increase it a bit?